tasks/eth/005-protocol-versions-ecotone/README.md

```diff
@@ -0,0 +1,211 @@
+Both versions are currently set to 3.1.0 (Regolith), as they were never updated since the deployment of `ProtocolVersions` for Canyon or Delta.
+[Task-003](../003-protocol-versions-delta/) is scheduled to update these versions to 5.0.0, and this task will then upgrade them to 6.0.0.
+
+## Approving the transaction
+
+### 1. Update repo and move to the appropriate folder for this rehearsal task:
+
+```
+cd superchain-ops
+git pull
+just install
+cd tasks/eth/005-protocol-versions-ecotone
+```
+
+### 2. Setup Ledger
+
+Your Ledger needs to be connected and unlocked. The Ethereum
+application needs to be opened on Ledger with the message "Application
+is ready".
+
+### 3. Simulate and validate the transaction
+
+Make sure your ledger is still unlocked and run the following.
+
+Remember that by default `just` is running with the address derived from
+`/0` (first nonce). If you wish to use a different account, run `just
+simulate [X]`, where X is the derivation path of the address
+that you want to use.
+
+You will see a "Simulation link" from the output.
+
+Paste this URL in your browser. A prompt may ask you to choose a
+project, any project will do. You can create one if necessary.
+
+Click "Simulate Transaction".
+
+We will be performing 3 validations and extract the domain hash and
+message hash to approve on your Ledger:
+
+1. Validate integrity of the simulation.
+2. Validate correctness of the state diff.
+3. Validate and extract domain hash and message hash to approve.
+
+#### 3.1. Validate integrity of the simulation.
+
+Make sure you are on the "Overview" tab of the tenderly simulation, to
+validate integrity of the simulation, we need to check the following:
+
+1. "Network": Check the network is mainnet.
+2. "Timestamp": Check the simulation is performed on a block with a
+   recent timestamp (i.e. close to when you run the script).
+3. "Sender": Check the address shown is your signer account. If not,
+   you will need to determine which “number” it is in the list of
+   addresses on your ledger. By default the script will assume the
+   derivation path is `m/44'/60'/0'/0/0`. By calling the script with
+   `just simulate 1` it will derive the address using
+   `m/44'/60'/1'/0/0` instead.
+
+#### 3.2. Validate correctness of the state diff.
+
+Now click on the "State" tab. Verify that:
+
+* For the `ProtocolVersions` Proxy at `0x8062AbC286f5e7D9428a0Ccb9AbD71e50d93b935` both the
+  recommended and required storage slots are updated from the encoded form of `3.1.0` to `6.0.0`.
+  * key `0x4aaefe95bd84fd3f32700cf3b7566bc944b73138e41958b5785826df2aecace0`
+    * before: `0x0000000000000000000000000000000000000003000000010000000000000000` (before exectuion of task-003) or
+    * before: `0x0000000000000000000000000000000000000005000000000000000000000000` (after exectuion of task-003)
+    * after : `0x0000000000000000000000000000000000000006000000000000000000000000`
+  * key `0xe314dfc40f0025322aacc0ba8ef420b62fb3b702cf01e0cdf3d829117ac2ff1a`
```

Comment:
Hm, I don't see this key in the simulation state changes, but I do see two `ConfigUpdated` events which look correct. Here's what I ran:
```
just install
forge clean
forge build
cd tasks/eth/005-protocol-versions-ecotone
export SIMULATE_WITHOUT_LEDGER=1
just simulate council
```

And here are the tenderly screenshots:

<img width="1162" alt="image" src="https://github.com/ethereum-optimism/superchain-ops/assets/17163988/149be0f3-5e49-416e-b643-12ea4e8a9dc0">

<img width="1551" alt="image" src="https://github.com/ethereum-optimism/superchain-ops/assets/17163988/ee381bda-f0bc-4b87-82e5-168f25763e7e">

Reply:
I see the trace in the README screen shot if I just run `just simulate`.

Reply:
Tenderly confirmed this was a bug, and it's now resolved!